libs/user/user.ts

```diff
@@ -372,6 +376,46 @@ class User {
     }
   }
 
+  /**
+   * @function importAndSaveCommitments should coverage the import commitments flow.
+   * - Should import a file with commitments.
+   * - Verify if all the commitments in the list of imported commitments are of the
+   * ICommitment type (This verification is within importCommitments function).
+   * - Verify if all the commitments belongs to the user compressedZkpPublicKey.
+   * - If all verifications pass, should send the commitments to the client to be saved
+   *  in the database.
+   * @param pathToExport the path to export the file.
+   * @param fileName the name of the file.
+   * @param compressedZkpPublicKey the key derivated from user mnemonic.
+   * @author luizoamorim
+   */
+  async importAndSaveCommitments(
+    pathToExport: string,
+    fileName: string,
+    compressedZkpPublicKey: string,
+  ) {
+    const listOfCommitments: Commitment[] | Error = await importCommitments(
+      `${pathToExport}${fileName}`,
+    );
+
+    if (listOfCommitments instanceof Error) {
+      logger.error(listOfCommitments);
+      return;
+    }
+
+    const isCommitmentsFromMnemonicReturn = await isCommitmentsFromMnemonic(
+      listOfCommitments,
+      compressedZkpPublicKey,
+    );
+
+    if (!isCommitmentsFromMnemonicReturn) {
+      logger.error(ERROR_COMMITMENT_NOT_MATCH_MNEMONIC);
+      return;
+    }
+
+    this.client.saveCommitments(listOfCommitments);
```

Comment:
Summary of changes in user:

- User method could just be: `importCommitments` - to be aligned with the export
- Method should receive `options` object which you should validate using Joi
- Then you could have 2 steps: a) validation + b) client call
- In the validation, validate both the commitment structure (like isra suggested for example), you can also validate the compressed..Key here
- Then call client. Client method could be named `insertCommitments` - more aligned with CRUD.